guides/plugins/plugins/framework/rule/add-custom-rules.md

```diff
@@ -199,3 +203,74 @@ The last step is, creating a template for our condition. We will create a new fi
+
+Rules in the shopware administration are now aware of where they're assigned to. This means that a user won't be able to add a rule to a promotion when the rule contains the condition "Cart amount". This also works the other way around. The user won't be able to add a the condition "Cart amount" when the rule is already assigned to a promotion.
+
+It's possible to define where rules are able to be assigned to. We'll show you how.
+
+### Defining restrictions
+
+We previously added our condition inside the `ruleConditionDataProviderService`. And this is the place where we define restrictions for our rule. The goal is that the user won't be able to add rules to advanced prices of a product.
+
+First we get the already existing definition for the rule relation like so:
+
+{% code title="<plugin root>src/Resources/app/administration/src/app/decorator/condition-type-data-provider.decorator.js" %}
+```javascript
+// Inside the addServiceProviderDecorator method
+const restrictions = ruleConditionService.getAwarenessConfigurationByAssignmentName('productPrices');
+```
+{% endcode %}
+
+{% hint style="info" %}
+All possible relations can be found in `Shopware\Core\Content\Rule\RuleDefinition`;
+{% endhint %}
+
+Now we want to add our own configuration. Therefore we import the `merge` function from `Shopware.Utils.object` then we will call the `addAwarenessConfiguration` method.
+
+{% code title="<plugin root>src/Resources/app/administration/src/app/decorator/condition-type-data-provider.decorator.js" %}
+```javascript
+Shopware.Application.addServiceProviderDecorator('ruleConditionDataProviderService', (ruleConditionService) => {
+    // Your newly added conditions is here
+
+    const restrictions = ruleConditionService.getAwarenessConfigurationByAssignmentName('productPrices');
+
+    ruleConditionService
+        .addAwarenessConfiguration('productPrices', merge(restrictions, {
+            notEquals: [
+                'lunar_eclipse',
+            ],
+            equalsAny: [], // ignore property if not needed
+            snippet: 'sw-restricted-rules.restrictedAssignment.lunarEclipse',
+        }),
+});
+```
+{% endcode %}
+
+We'll explain what `notEquals` and `equalsAny` mean later. Now we just need a new snippet. You can [see here](../../administration/adding-snippets.md) how to add snippets to shopware 6. The path to the directory is: `<plugin root>src/Resources/app/administration/src/app/snippet`.
+
+After you've added the snippet you successfully definded resctrictions for you custom condition. If you now try to assign a rule with your condition to advanced prices you should see that it's not possible and the rule is disabled.
+
+What do `notEquals` and `equalsAny` actually mean. With these two properties you can define that rules, you want to assign to a specific relation i.e. `productPrices`, need to have at least one condition that is inside `equalsAny` if specified or should not have any condition inside of `notEquals` if specified.
```

Comment:
Own opinion: Leave the `if specified` out, its stops my read flow :laughing: But this is optional.

Reply:
Is out.